facia/app/views/fragments/brandedFrontBody.scala.html

```diff
@@ -0,0 +1,61 @@
+@(faciaPage: model.PressedPage)(implicit request: RequestHeader)
+@import common.Edition
+@import common.commercial.ContainerModel
+@import model.{PaidContent, Sponsored}
+@import views.support.RenderClasses
+
+@faciaPage.branding(Edition(request)) match {
+
+    case Some(branding) => {
```

Comment:
if we have no branding here then we call the `frontBody` fragment. Would it be worth putting the branded frontBody code into a separate fragment too?